### PR TITLE
W-17027111-SF-Connector-Time-field-value-for-operations-KS

### PR DESCRIPTION
**Pull request by Kabilan-s-cx**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: salesforce/10.22/modules/ROOT/pages/salesforce-connector-reference.adoc [hunk 1]
```diff
@@ -731,7 +731,8 @@ Source that subscribes to a topic. First, the topic must be published and after
 * <<upsert>>
 * <<upsertMetadata>>
 
-
+[NOTE]
+If you set a `LocalDate` for a field in the payload, the input time gets automatically converted to `GMT` or the server’s timezone based on the `GregorianCalendar`. This conversion impacts date calculations or comparisons in Salesforce connector operations, as the `LocalDate` is adjusted to align with the server’s timezone or defaults to GMT.
```
Comment (JennyHajee) [inline salesforce/10.22/modules/ROOT/pages/salesforce-connector-reference.adoc]:
```suggestion
If you set a `LocalDate` for a field in the payload, the input time gets automatically converted to `GMT` or the server’s time zone based on the `GregorianCalendar`. This conversion impacts date calculations or comparisons in Salesforce connector operations, as the `LocalDate` is adjusted to align with the server’s time zone or defaults to GMT.
```

**Review by JennyHajee — DISMISSED** [review]

File: salesforce/10.22/modules/ROOT/pages/salesforce-connector-reference.adoc [hunk 2]
```diff
@@ -731,7 +731,8 @@ Source that subscribes to a topic. First, the topic must be published and after
 * <<upsert>>
 * <<upsertMetadata>>
 
-
```
Comment (sathya0) [inline salesforce/10.22/modules/ROOT/pages/salesforce-connector-reference.adoc]: move the note to before the list of operations

File: salesforce/10.22/modules/ROOT/pages/salesforce-connector-reference.adoc [hunk 3]
```diff
@@ -731,7 +731,8 @@ Source that subscribes to a topic. First, the topic must be published and after
...
+[NOTE]
+If you set a `LocalDate` for a field in the payload, the input time gets automatically converted to `GMT` or the server’s time zone based on the `GregorianCalendar`. This conversion impacts date calculations or comparisons in Salesforce connector operations, as the `LocalDate` is adjusted to align with the server’s time zone or defaults to GMT.
```
Comment (sathya0) [inline salesforce/10.22/modules/ROOT/pages/salesforce-connector-reference.adoc]:
```suggestion
If you set `LocalDate` for a field in the payload, the input time is automatically converted to `GMT` or the server’s time zone based on the Gregorian calendar. This conversion impacts date calculations or comparisons in Salesforce Connector operations, as `LocalDate` is adjusted to align with the server’s time zone or defaults to GMT.
```

File: salesforce/10.22/modules/ROOT/pages/salesforce-connector-reference.adoc [hunk 4]
```diff
@@ -2688,7 +2689,15 @@ Invokes any operation from an Apex class that is exposed as a REST web service.
 
 `queryParams: {AccountId : "a0w0E0000069MDxQAM" , ContactId: "0030E00000rRSJ6QAO" }` |  `#[payload]` |
 | Apex Class Name a| String | Apex class name. |  | x
-| Apex Class Method Name a| String | Apex class method name. |  | x
+| Apex Class Method Name a| String | Apex class method name. The method name format is like,
```
Comment (sathya0) [inline salesforce/10.22/modules/ROOT/pages/salesforce-connector-reference.adoc]:
```suggestion
| Apex Class Method Name a| String | Apex class method name. The method name format is:
```

File: salesforce/10.22/modules/ROOT/pages/salesforce-connector-reference.adoc [hunk 5]
```diff
@@ -2688,7 +2689,15 @@ Invokes any operation from an Apex class that is exposed as a REST web service.
 
 `queryParams: {AccountId : "a0w0E0000069MDxQAM" , ContactId: "0030E00000rRSJ6QAO" }` |  `#[payload]` |
 | Apex Class Name a| String | Apex class name. |  | x
-| Apex Class Method Name a| String | Apex class method name. |  | x
+| Apex Class Method Name a| String | Apex class method name. The method name format is like, 
+
+`ApexMethodName&#94;RestResourceURL&#94;HttpMethod&#94;OutputType&#94;InputParameterName1=ParameterType1,InputParameterName2=ParameterType2`
```
Comment (sathya0) [inline salesforce/10.22/modules/ROOT/pages/salesforce-connector-reference.adoc:2696]: add line break in between 2694 and 2695

**Review by sathya0 — APPROVED** [review]

**Pull request merged**